docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -35,20 +40,20 @@ proxy_service:
   kube_listen_addr: 0.0.0.0:3026
   # optional: set a different public address for kubernetes access
   kube_public_addr: kube.example.com:3026
+```
+
+You will also need to have a `kubernetes_service` section in your `teleport.yaml` file. This can be in your existing proxy, or a new separate teleport instance:
```

Comment:
```suggestion
You will also need to include a `kubernetes_service` section in your `teleport.yaml` file. This can be in your existing proxy or a separate Teleport instance:
```